Arithmetic on the Command Line for 12-cmdline.md suggestions (#842)
swcarpentry/python-novice-inflammation: _episodes/12-cmdline.md

> ## Arithmetic on the Command Line
>

> > Write a command-line program that does addition and subtraction (you could also try multiplication and division operators):

Reviewer: Since the solution contains multiplication and division, I suggest we include it along with addition and subtraction in the main challenge description. Also, since the arithmetic only works with 2 numbers, we should probably specify that as well.
```suggestion
> Write a command-line program that performs addition, subtraction,
 multiplication, or division with two numbers:
```

Reviewer 2: I agree with Lauren's suggestion to improve this change and I'd like to propose further improvement:
```suggestion
> Write a Python program that adds, subtracts, multiplies, or divides two numbers provided on the command line:
```

Reviewer: I like the version by @maxim-belkin .

> >     assert len(sys.argv) == 4, 'Need exactly 3 arguments'
> >
> >     operator = sys.argv[1]
> >     assert operator in ['--add', '--subtract', '--multiply', '--divide'], \

> > >         'Operator is not one of add, subtract, multiply, or divide: bailing out'

Reviewer: To prevent confusion, do you think we should have the dashes in the error message as well?
```suggestion
> >         'Operator is not one of --add, --subtract, --multiply, or --divide: bailing out'
```
